docs/2.2/generated/torch.compile.html

```diff
@@ -498,7 +498,7 @@ <h1>torch.compile<a class="headerlink" href="#torch-compile" title="Permalink to
 <li><p>”inductor” is the default backend, which is a good balance between performance and overhead</p></li>
 <li><p>Non experimental in-tree backends can be seen with <cite>torch._dynamo.list_backends()</cite></p></li>
 <li><p>Experimental or debug in-tree backends can be seen with <cite>torch._dynamo.list_backends(None)</cite></p></li>
-<li><p>To register an out-of-tree custom backend: <a class="reference external" href="https://pytorch.org/docs/main/compile/custom-backends.html">https://pytorch.org/docs/main/compile/custom-backends.html</a></p></li>
```

Comment:
https://pytorch.org/docs/main/compile/custom-backends.html and https://pytorch.org/docs/main/compile/custom-backends.html are dead links